### PR TITLE
Fixes to bootstrapping Akaros

### PR DESCRIPTION
**Pull request by mitthu**
### Summary
* Remove use of `AKAROS_XCC_ROOT`
* Add support for `AKAROS_TOOLCHAINS`
* Clean up GETTING_STARTED.md document
* Run the init script when using `make defconfig`

### CODE REVIEW

**Review by brho — COMMENTED** [review]
Two minor things for the commit message of the first commit (Remove):
- First letter is capitalized, unless it's a category.  So Remove, not remove.  A category is something like "usb: foo"
- Should explain why it's OK to remove this.  e.g. "turns out it was only used to compute AKAROS_SYSROOT, which we can compute directly with -print-sysroot" or whatever.  other than explaining to other people what we are doing, it'll also be a reminder to ourselves or to any future bug hunters.  perhaps we were wrong to remove this, and since we knew why we removed it and what we were thinking, we'll be better able to diagnose any issues.

File: tools/compilers/gcc-glibc/Makefile [hunk 1]
```diff
@@ -8,8 +8,10 @@ GMP_VERSION := 5.1.3
 MPFR_VERSION := 3.1.2
 MPC_VERSION := 1.0.2
 
-# Defines where to install the toolchain
--include Makelocal
+# Define where to install the toolchain
```
Comment (brho) [inline tools/compilers/gcc-glibc/Makefile]: can you add a check to make sure AKAROS_TOOLCHAINS is not empty?  there are example checks in tools/Makefrag.

also, the commit needs an (XCC) tag.  I'd also put a line in the commit message that says "Delete your toolchain Makelocal and set AKAROS_TOOLCHAINS in your environment.  Rebuild your toolchain"  i'd also clarify the difference between AKAROS_TOOLCHAINS and the old AKAROS_XCC_ROOT.  XCC_ROOT was a particular toolchain (e.g. x86_64).  AKAROS_TOOLCHAINS is the directory for all toolchains, e.g. riscv, x86_64.  So TOOLCHAINS is the parent of the old XCC_ROOT.

Comment (mitthu) [reply tools/compilers/gcc-glibc/Makefile]: On it!

Comment (brho) [reply tools/compilers/gcc-glibc/Makefile]: still need a check for AKAROS_TOOLCHAINS being set or not.  can use something like the one in tools/Makefrag

Comment (brho) [reply tools/compilers/gcc-glibc/Makefile]: and i'd squash this fix into the "  Add AKAROS_TOOLCHAINS (XCC)" commit

Comment (mitthu) [reply tools/compilers/gcc-glibc/Makefile]: The latest commit fixes it.

**Pull request merged**